### PR TITLE
Did some error handling

### PR DESCRIPTION
**Pull request by aniketbiswas21**
Error handling in Login page.

### CODE REVIEW

File: src/redux/reducers/authReducer.js [hunk 1]
```diff
@@ -34,6 +40,11 @@ export default (state = defaultState, action) => {
         ...state,
         loading: true,
       };
+    case RESET_STATE:
+      return {
+        ...state,
+        error: null,
```
Comment (jsparmani) [inline src/redux/reducers/authReducer.js:46]: use `return defaultState` directly.

**Comment by jsparmani** [comment]
Also, the `Alert` used is not looking good. Try using `message` / `Notification`.

File: src/pages/Login.js [hunk 2]
```diff
@@ -30,19 +29,8 @@ const Login = ({ auth: { error }, resetState }) => {
...
-                showIcon
-                closable={true}
-                onClose={() => {
-                  resetState();
-                }}
-              />
-            </Col>
-          )}
+          {error &&
+            message.info('Login Failed. Please try again.', 2, resetState())}
```
Comment (jsparmani) [inline src/pages/Login.js:33]: why using `message.info`. I think `message.error` is required.

**Comment by aniketbiswas21** [comment]
Okay.

**Comment by aniketbiswas21** [comment]
Done @jsparmani

File: src/redux/reducers/authReducer.js (hunk 1 above)
Comment (jsparmani) [reply src/redux/reducers/authReducer.js]: @aniketbiswas21 Has this been done?

File: src/pages/Login.js (hunk 2 above)
Comment (jsparmani) [reply src/pages/Login.js:33]: @aniketbiswas21 Has this been done?

**Pull request merged**